prow/github/util/util.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package github
+
+import "k8s.io/test-infra/prow/github"
+
+type labelLister interface {
+	GetIssueLabels(org, repo string, number int) ([]github.Label, error)
+}
+
+// HasLabel checks if a label is applied to a pr.
+func HasLabel(c labelLister, org, repo string, num int, label string) (bool, error) {
```

Comment:
@spxtr want to review this expansion of the core prow api?

Reply:
Much appreciated, I was not super clear on how to do this since `util` packages always kind of suck

Reply:
I agree, `util` packages suck. I think for this one you can just do a `GetIssue` followed by [`Issue.HasLabel`](https://godoc.org/github.com/kubernetes/test-infra/prow/github#Issue.HasLabel). No need for the util package, and it will only add a couple lines elsewhere.